src/components/MoviesList/MoviesList.jsx

```diff
@@ -1,7 +1,7 @@
 import './MoviesList.scss';
 import { MovieCard } from '../MovieCard';
 
-export const MoviesList = ({ movies }) => (
+export const MoviesList = ({ movies, filterBy }) => (
```

Comment:
```suggestion
export const MoviesList = ({ movies }) => (
```

Looks like you don't use `filterBy` in this component. So why we need to pass it?